app/controllers/users/accounts_controller.rb

```diff
@@ -0,0 +1,20 @@
+# -*- encoding : utf-8 -*-
+class Users::AccountsController < ApplicationController
+  before_filter :check_pro_pricing_enabled
+
+  def show
+  end
+
+  def edit
+  end
+
+  def update
+    redirect_to users_account_path
+  end
+
+  private
+
+  def check_pro_pricing_enabled
+    raise ActiveRecord::RecordNotFound unless feature_enabled?(:pro_pricing)
+  end
+end
```

Comment:
probably worth having a stub filter method or `# TODO` in here as a visual reminder that these pages will be restricted to the owning user (makes the purpose of the controller more obvious at a glance)